index.js

```diff
@@ -1,25 +1,49 @@
 // @see https://github.com/nodejs/node/blob/master/doc/guides/doc-style-guide.md
 
-"use strict";
+import remarkPresetLintRecommended from "remark-preset-lint-recommended";
+import remarkLintBlockquoteIndentation from "remark-lint-blockquote-indentation";
+import remarkLintCheckboxCharacterStyle from "remark-lint-checkbox-character-style";
+import remarkLintCheckboxContentIndent from "remark-lint-checkbox-content-indent";
+import remarkLintCodeBlockStyle from "remark-lint-code-block-style";
+import remarkLintDefinitionSpacing from "remark-lint-definition-spacing";
+import remarkLintFencedCodeFlag from "remark-lint-fenced-code-flag";
+import remarkLintFencedCodeMarker from "remark-lint-fenced-code-marker";
+import remarkLintFileExtension from "remark-lint-file-extension";
+import remarkLintFinalDefinition from "remark-lint-final-definition";
+import remarkLintFirstHeadingLevel from "remark-lint-first-heading-level";
+import remarkLintHeadingStyle from "remark-lint-heading-style";
+import remarkLintListItemIndent from "remark-lint-list-item-indent";
+import remarkLintMaximumLineLength from "remark-lint-maximum-line-length";
+import remarkLintNoConsecutiveBlankLines from "remark-lint-no-consecutive-blank-lines";
+import remarkLintNoFileNameArticles from "remark-lint-no-file-name-articles";
+import remarkLintNoFileNameConsecutiveDashes from "remark-lint-no-file-name-consecutive-dashes";
+import remarkLintNofileNameOuterDashes from "remark-lint-no-file-name-outer-dashes";
+import remarkLintNoHeadingIndent from "remark-lint-no-heading-indent";
+import remarkLintNoMultipleToplevelHeadings from "remark-lint-no-multiple-toplevel-headings";
+import remarkLintNoShellDollars from "remark-lint-no-shell-dollars";
+import remarkLintNoTableIndentation from "remark-lint-no-table-indentation";
+import remarkLintNoTabs from "remark-lint-no-tabs";
+import remarkLintNoTrailingSpaces from "remark-lint-no-trailing-spaces";
+import remarkLintNodejsLinks from "./remark-lint-nodejs-links.js";
+import remarkLintNodejsYamlComments from "./remark-lint-nodejs-yaml-comments.js";
+import remarkLintProhibitedStrings from "remark-lint-prohibited-strings";
```

Comment:
Does this need the new release, or is it OK as-is?

Reply:
Should be OK as-is.